### ISSUE TITLE
[ENHANCEMENT] [v1.2] Display Alt Text - Identify missing alt attribute

### ISSUE THREAD

**Issue by ntibbles** [ntibbles/equa11y#88]
Currently, the extension infers an empty alt text when the alt attribute may be missing. Include a check to make sure the alt attribute exists.
